extensions/wikia/LanguageWikisIndex/styles/language-wikis-index.scss

```diff
@@ -0,0 +1,137 @@
+@import 'skins/oasis/css/core/layout';
+@import 'skins/oasis/css/core/breakpoints-variables';
+@import 'extensions/wikia/DesignSystem/node_modules/design-system/dist/scss/wds-variables/index';
+
+$language-wikis-index-header-background-color: #006CB0;
+$language-wikis-index-header-background-height: 105px;
+// Keep in sync with ThemeSettings::BACKGROUND_HEIGHT
```

Comment:
We're not displaying community's background so there is no need to keep it in sync.